<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugLog</class>
 <widget class="QWidget" name="DebugLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>553</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="VerticalOnlyScroller" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>533</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
       <item>
        <widget class="QGroupBox" name="debugLogGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Debug Log</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="cbAutoReload">
              <property name="text">
               <string>Auto-reload</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="cbAutoScroll">
              <property name="text">
               <string>Auto-scroll</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbFilters"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QListView" name="debugLog"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VerticalOnlyScroller</class>
   <extends>QScrollArea</extends>
   <header>src/widget/form/settings/verticalonlyscroller.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
